### PR TITLE
Add args validation to E2E tests

### PR DESCRIPTION
**Pull request by ramo-j**
Add args validation to E2E tests

Also, correction to aws subnets - they're AWS subnet IDs, not actual network addresses with masks.

### CODE REVIEW

File: tests/e2e/aws_disk_forensics.py [hunk 1]
```diff
@@ -96,13 +96,13 @@
...
-    ['volumes', 'Comma separated list of EBS volume IDs (e.g. vol-xxxxxxxx).', None],
-    ['aws_bucket', 'AWS bucket for image storage.', None],
-    ['gcp_bucket', 'GCP bucket for image storage.', None],
-    ['--subnet', 'AWS subnet to copy instances from, required if there is no default subnet in the volume region.', None],
+  'args': [
+    ['aws_region', 'AWS region containing the EBS volumes.', None, {'format': 'aws_region'}],
+    ['gcp_zone', 'Destination GCP zone in which to create the disks.', None, {'format': 'gcp_zone'}],
+    ['volumes', 'Comma separated list of EBS volume IDs (e.g. vol-xxxxxxxx).', None, {'format': 'regex', 'comma_separated': True, 'regex': '^vol-[0-9a-f]{8,17}$'}],
+    ['aws_bucket', 'AWS bucket for image storage.', None, {'format': 'regex', 'comma_separated': False, 'regex': '^(s3:\/\/)?[0-9a-z][-\\.0-9a-z]{1,61}[0-9a-z]$'}],
+    ["gcp_bucket", "GCP bucket for image storage.", None, {'format': 'regex', 'comma_separated': False, 'regex': '^(gs:\/\/)?[0-9a-z][-\\.0-9a-z_]{1,61}[0-9a-z]$'}],
```
Comment (sydp) [inline tests/e2e/aws_disk_forensics.py:104]: change in type of quotes here?

**Review by sydp — DISMISSED** [review]
LGTM (w/ a minor nit)

File: tests/e2e/aws_disk_forensics.py (hunk 1 above)
Comment (ramo-j) [reply tests/e2e/aws_disk_forensics.py]: Copy paste fail. Fixed.

**Review by sydp — APPROVED** [review]
Lgtm

**Pull request merged**